test/irb/test_ruby_lex.rb

```diff
@@ -33,7 +33,7 @@ def teardown
     def calculate_indenting(lines, add_new_line)
       lines = lines + [""] if add_new_line
       last_line_index = lines.length - 1
-      byte_pointer = lines.last.length
+      byte_pointer = lines.last.bytesize
```

Comment:
The test seems to capture the issue even without this change?

Reply:
Yes. This change don't affect the added test. I just fixed it because I found it doing wrong with byte_pointer.